src/tasks/containers/TaskPage.tsx

```diff
@@ -57,6 +59,7 @@ class TaskPage extends PureComponent<Props> {
 
   public render(): JSX.Element {
     const {newScript, taskOptions} = this.props
+    const showNewTasksUI = isFlagEnabled('tasksUiEnhancements')
```

Comment:
nitpick: maybe call this `shouldShowNewTasksUI`, since `showNewTasksUI` is verby and might be confused for a function name